### ISSUE TITLE
ReplayMod

### ISSUE THREAD

**Issue by KaaayKaay** [Keksuccino/FancyMenu#810]
You cannot edit or copy the buttons from replaymod at all, even in the editor the buttons still show up as if you were in a normal screen and are fully interactable, the only thing that can be edited is the replaymod button on the title screen.

Latest fancymenu version for 1.20.1
Latest replaymod version for 1.20.1

**Comment by Keksuccino** [comment]
Yeah that's on purpose. I manually disabled customizations for that mod, because it is super incompatible.
